flink-cdc-common/src/main/java/org/apache/flink/cdc/common/configuration/ConfigurationUtils.java

```diff
@@ -130,7 +130,9 @@ static Duration convertToDuration(Object o) {
     }
 
     static String convertToString(Object o) {
-        if (o.getClass() == String.class) {
+        if (o == null) {
```

Comment:
Maybe we can bring the check to `convertValue` function? Seems other `convertToXXX` functions also share this problem.

Reply:
you are right, just done it.

Reply:
The inner null check seems redundant now.

Reply:
> The inner null check seems redundant now.

@loserwang1024 WDYT?

Reply:
> > The inner null check seems redundant now.

Maybe a java.lang.IllegalArgumentException with description is better than NullPointerException?